### ISSUE TITLE
Where with bool or bool? is compiled as "true" or "false" rather than "0" or "1"

### ISSUE THREAD

**Issue by benmccallum** [sqlkata/querybuilder#396]
**Repro**
Go here: https://sqlkata.com/playground/sqlserver
Paste this:
```
bool? lol = true;
bool lol2 = true;
var query = new Query("Users").Where("lol", lol).Where("lol2", lol2);
```

SQL Server doesn't support true/false, only 0 or 1

**Comment by benmccallum** [comment]
Hmm, I believe maybe we're suppose to use `WhereTrue` or `WhereFalse`... Not 100% clear though. Could do with an example on the Where page of docs.

**Comment by benmccallum** [comment]
I swear I've had this working before though... Could you confirm if it's a bug or not?

**Comment by benmccallum** [comment]
A workaround is to do a sneaky `Convert.ToInt16(lol)` in there, but this is a bit of a nasty regression if that's what it is. Could bite a lot of people.

**Comment by benmccallum** [comment]
Maybe there could be a `.WhereBooleanIs(true/false)` extension method?

**Comment by ahmad-moussawi** [comment]
I think this was solved by #510 
check https://sqlkata.com/release/v2.3.8#whereboolean-is-now-identical-to-wheretrue-and-wherefalse
So it can be closed now
